packages/KTable/KTable.vue

```diff
@@ -93,6 +117,19 @@ table.k-table {
       text-align: left;
       font-size: var(--KTableHeaderSize, var(--type-sm, type(sm)));
       font-weight: 500;
+      cursor: pointer;
```

Comment:
this is a little confusing for when a column cannot be sorted, so we probably shouldn't put a cursor for all tables and all header cells. Not sure how it should work right now as click the headers in the vuepress docs don't do anything

Reply:
yeah not sure what is a good example for the vue docs since the sort logic is left up to the parent implementation for it to actually do anything

Reply:
we should add an example in the docs of a parent component using ktable sort